samples/SocketsSample/wwwroot/hubs.html

```diff
@@ -104,7 +104,7 @@ <h4>Private Message</h4>
     connectButton.disabled = true;
     disconnectButton.disabled = false;
     console.log('http://' + document.location.host + '/' + hubRoute);
-    connection = new signalR.HubConnection(hubRoute, logger, { transport: transportType, logger: logger });
+    connection = new signalR.HubConnection(hubRoute, { transport: transportType, logging: logger });
```

Comment:
Copycat https://github.com/aspnet/SignalR/pull/1158/files#diff-c80b6b1153b8f1588d86994e37bb3c7fR107

Reply:
Mine's better, I fixed all of them :P.